extensions/rich_text_components/components.py

```diff
@@ -150,3 +150,16 @@ def validate(cls, value_dict):
         video_id = value_dict['video_id-with-value']
         if len(video_id) != 11:
             raise Exception('Video id length is not 11')
+
+
+class Audio(BaseRteComponent):
+    """Class fro Audio component."""
+
+    @classmethod
+    def validate(cls, value_dict):
+        """Validates Audio component."""
+        super(Audio, cls).validate(value_dict)
+        filename_re = r'^[A-Za-z0-9+/]*\.((mp3)|(amr)|(wave)|(aac))$'
```

Comment:
I think we are expecting to have `_` in the file name, right? Also we are going to have an expected length and pattern for the file name so can we validate this?